week-1/tic-tac-toe/launch_game.rb

```diff
@@ -0,0 +1,21 @@
+require "./Classes/Bot.rb"
```

Comment:
Crear branches para guardar versiones propuestas del software

Create branches to saver software's proposed versions.